src/main/java/org/opensearch/security/tools/PasswordSetup.java

```diff
@@ -0,0 +1,680 @@
+                        System.out.println("   * Make also sure that your keystore or PEM certificate is a client certificate (not a node certificate) and configured properly in opensearch.yml");
+                        System.out.println("   * If this is not working, try running securityadmin.sh with --diagnose and see diagnose trace log file)"); 
+                        System.out.println("   * Add --accept-red-cluster to allow securityadmin to operate on a red cluster.");
+
+                        return (-1);
+                    }
+                    
+                    Thread.sleep(3000);
+                    continue;
+                }
+            }
+
+            final boolean timedOut = chResponse.isTimedOut();
+
+            if (!acceptRedCluster && timedOut) {
+                System.out.println("ERR: Timed out while waiting for a green or yellow cluster state.");
+                System.out.println("   * Try running securityadmin.sh with -icl (but no -cl) and -nhnv (If that works you need to check your clustername as well as hostnames in your TLS certificates)");
+                System.out.println("   * Make also sure that your keystore or PEM certificate is a client certificate (not a node certificate) and configured properly in opensearch.yml");
+                System.out.println("   * If this is not working, try running securityadmin.sh with --diagnose and see diagnose trace log file)"); 
+                System.out.println("   * Add --accept-red-cluster to allow securityadmin to operate on a red cluster.");
+                return (-1);
+            }
+
+            System.out.println("Clustername: " + chResponse.getClusterName());
+			System.out.println("Clusterstate: " + chResponse.getStatus());
+			System.out.println("Number of nodes: " + chResponse.getNumberOfNodes());
+			System.out.println("Number of data nodes: " + chResponse.getNumberOfDataNodes());
+
+            Scanner sc = new Scanner(System.in);
+            ArrayList<String> users = new ArrayList<String>(Arrays.asList("admin", "kibanaserver", "kibanaro", "logstash", "readall", "snapshotrestore"));
+
+            System.out.println("\n\nBeginning Password Setup");
+
+            for (String user: users) {
+                boolean isWeakPassword = true;
+                while(isWeakPassword) {
+                    try {
+                        System.out.println("\nEnter password for " + user + ": ");
+                        String password = sc.nextLine();
+                        setPasswordSingleUser(user, lowLevelClient, password);
+                        isWeakPassword = false;
+                    } catch(ResponseException e) {
+                        System.out.println("A password must be at least 8 characters long and contain at least one uppercase letter, one lowercase letter, one digit, and one special character.");
```

Comment:
It should log the message in `plugins.security.restapi.password_validation_error_message`